data_platform/ingestion/sync_bluesky.py

```
@@ -249,145 +245,95 @@ def run_sync_tasks(
     print(
-        f"sync_records: wrote {total_rows} rows to {output_dir} (status={metadata['sync_status']})"
+        f"sync_records: wrote {metadata['row_count']} rows to {output_dir} "
+        f"(status={metadata['sync_status']})"
+    )
+
+
+BLUESKY_SYNC_SPEC = SyncPlatformSpec(
+    platform="bluesky",
+    storage_cls=BlueskyStorageManager,
+    entity_label="keywords",
+    init_client=lambda: init_bluesky_client(),
```

Ruff (0.16.3)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)